src/main/java/org/apache/bcel/classfile/Code.java

```diff
@@ -247,6 +247,20 @@ public LineNumberTable getLineNumberTable() {
         return null;
     }
 
+    /**
+     * Finds the first attribute of {@link StackMap} instance.
+     * @return StackMap of Code, if it has one, else null.
+     * @since 6.7.1
+     */
+    public StackMap getStackMap() {
```

Comment:
Should this be called getFirstStackMap()?

Reply:
There may be at most one StackMapTable attribute in the attributes table of a Code attribute. https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.7.4

Reply:
Ah, ok, then it's the comment that needs updating IMO, as "first" makes it sound like there can be more than one.

Reply:
Yes. Removed "first".